### PR TITLE
Plants 106 master

### PR DESCRIPTION
**Pull request by gnaamati**
## Description

Plants tree and allowed species for 106

**Related JIRA tickets:**
- ENSCOMPARASW-XXXX

## Overview of changes
_Give details of what changes were required to solve the problem. Break into sections if applicable._

#### Change 1
- _detail 1.1_

#### Change 2
- _detail 2.1_

## Testing
_How was this tested? Have new unit tests been included?_

## Notes
_Optional extra information._

### CODE REVIEW

File: conf/plants/species_tree.topology.nw [hunk 1]
```diff
@@ -210,7 +227,7 @@
           (
            quercus_lobata,
            juglans_regia,
-           corylus_avellana,
+           corylus avellana,
```
Comment (JAlvarezJarreta) [inline conf/plants/species_tree.topology.nw:230]: It should be `corylus_avellana`, so please revert this change.

**Review by JAlvarezJarreta — CHANGES_REQUESTED** [review]

File: conf/plants/species_tree.topology.nw (hunk 1 above)
Comment (gnaamati) [reply conf/plants/species_tree.topology.nw:230]: Can't believe that happened again, will fix!

Comment (gnaamati) [reply conf/plants/species_tree.topology.nw]: Fixed it

File: conf/plants/species_tree.topology.nw [hunk 2]
```diff
@@ -55,32 +62,92 @@
...
                triticum_turgidum,
+               triticum_aestivum_arinalrfor,
+	       triticum_aestivum_jagger,
+	       triticum_aestivum_julius,
+	       triticum_aestivum_lancer,
+	       triticum_aestivum_landmark,
+	       triticum_aestivum_mace,
+	       triticum_aestivum_mattis,
+	       triticum_aestivum_norin61,
+	       triticum_aestivum_stanley,
```
Comment (JAlvarezJarreta) [inline conf/plants/species_tree.topology.nw:84]:
```suggestion
               triticum_aestivum_jagger,
               triticum_aestivum_julius,
               triticum_aestivum_lancer,
               triticum_aestivum_landmark,
               triticum_aestivum_mace,
               triticum_aestivum_mattis,
               triticum_aestivum_norin61,
               triticum_aestivum_stanley,
```

File: conf/plants/species_tree.topology.nw [hunk 3]
```diff
@@ -55,32 +62,92 @@
...
                 triticum_turgidum_A,
+                triticum_aestivum_arinalrfor_A,
+	        triticum_aestivum_jagger_A,
+	        triticum_aestivum_julius_A,
+	        triticum_aestivum_lancer_A,
+	        triticum_aestivum_landmark_A,
+	        triticum_aestivum_mace_A,
+	        triticum_aestivum_mattis_A,
+	        triticum_aestivum_norin61_A,
+	        triticum_aestivum_stanley_A,
```
Comment (JAlvarezJarreta) [inline conf/plants/species_tree.topology.nw:100]:
```suggestion
                triticum_aestivum_jagger_A,
                triticum_aestivum_julius_A,
                triticum_aestivum_lancer_A,
                triticum_aestivum_landmark_A,
                triticum_aestivum_mace_A,
                triticum_aestivum_mattis_A,
                triticum_aestivum_norin61_A,
                triticum_aestivum_stanley_A,
```

File: conf/plants/species_tree.topology.nw [hunk 4]
```diff
@@ -55,32 +62,92 @@
...
+               triticum_turgidum_B,
+               triticum_aestivum_arinalrfor_B,
+	       triticum_aestivum_jagger_B,
+	       triticum_aestivum_julius_B,
+	       triticum_aestivum_lancer_B,
+	       triticum_aestivum_landmark_B,
+	       triticum_aestivum_mace_B,
+	       triticum_aestivum_mattis_B,
+	       triticum_aestivum_norin61_B,
+	       triticum_aestivum_stanley_B,
```
Comment (JAlvarezJarreta) [inline conf/plants/species_tree.topology.nw:118]:
```suggestion
               triticum_aestivum_jagger_B,
               triticum_aestivum_julius_B,
               triticum_aestivum_lancer_B,
               triticum_aestivum_landmark_B,
               triticum_aestivum_mace_B,
               triticum_aestivum_mattis_B,
               triticum_aestivum_norin61_B,
               triticum_aestivum_stanley_B,
```

File: conf/plants/species_tree.topology.nw [hunk 5]
```diff
@@ -55,32 +62,92 @@
...
+               triticum_turgidum_D,
+               triticum_aestivum_arinalrfor_D,
+	       triticum_aestivum_jagger_D,
+	       triticum_aestivum_julius_D,
+	       triticum_aestivum_lancer_D,
+	       triticum_aestivum_landmark_D,
+	       triticum_aestivum_mace_D,
+	       triticum_aestivum_mattis_D,
+	       triticum_aestivum_norin61_D,
+	       triticum_aestivum_stanley_D,
```
Comment (JAlvarezJarreta) [inline conf/plants/species_tree.topology.nw:137]:
```suggestion
               triticum_aestivum_jagger_D,
               triticum_aestivum_julius_D,
               triticum_aestivum_lancer_D,
               triticum_aestivum_landmark_D,
               triticum_aestivum_mace_D,
               triticum_aestivum_mattis_D,
               triticum_aestivum_norin61_D,
               triticum_aestivum_stanley_D,
```

File: conf/plants/species_tree.topology.nw [hunk 6]
```diff
@@ -55,32 +62,92 @@
...
                triticum_turgidum_U,
+               triticum_aestivum_arinalrfor_U,
+	       triticum_aestivum_jagger_U,
+	       triticum_aestivum_julius_U,
+	       triticum_aestivum_lancer_U,
+	       triticum_aestivum_landmark_U,
+	       triticum_aestivum_mace_U,
+	       triticum_aestivum_mattis_U,
+	       triticum_aestivum_norin61_U,
+	       triticum_aestivum_stanley_U,
```
Comment (JAlvarezJarreta) [inline conf/plants/species_tree.topology.nw:150]:
```suggestion
               triticum_aestivum_jagger_U,
               triticum_aestivum_julius_U,
               triticum_aestivum_lancer_U,
               triticum_aestivum_landmark_U,
               triticum_aestivum_mace_U,
               triticum_aestivum_mattis_U,
               triticum_aestivum_norin61_U,
               triticum_aestivum_stanley_U,
```

**Review by JAlvarezJarreta — CHANGES_REQUESTED** [review]
Minor indentation issues with tabs that should be addressed for consistency across the document. Otherwise, it seems good to go :+1:

**Review by JAlvarezJarreta — APPROVED** [review]

**Review by dthybert — APPROVED** [review]
